tools/cdk-build-tools/package.json

```diff
@@ -35,32 +35,32 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/fs-extra": "^9.0.11",
-    "@types/jest": "^26.0.22",
-    "@types/semver": "^7.3.5",
+    "@types/jest": "^26.0.23",
     "@types/yargs": "^16.0.1",
+    "@types/semver": "^7.3.5",
```

Comment:
Ordering was changed here too so it isn't alphabetical

Reply:
Fixed